servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/TimeoutPublisher.java

```diff
@@ -73,6 +72,11 @@
         this.restartAtOnNext = restartAtOnNext;
     }
 
+    @Override
```

Comment:
the timeout operators are exceptional in that they require an executor to execute the timeout callback, so these operators will need to retain an `Executor`, but they should be able todo that as `private final` fields. fine to defer to followup PR if that is where you intended to do more cleanup.

Reply:
Yes, this will need to be cleaned up when `SignalOffloader` is removed.